service/frontend/service.go

```diff
@@ -102,7 +103,7 @@ func (s *Service) Start() {
 	}
 	visibility = persistence.NewVisibilityPersistenceClient(visibility, base.GetMetricsClient(), log)
 
-	history, err := persistence.NewCassandraHistoryPersistence(p.CassandraConfig.Hosts,
+	history, err := cassandra.NewCassandraHistoryPersistence(p.CassandraConfig.Hosts,
```

Comment:
Remove the Cassandra part in method name now as it's in cassandra package. We can change it to  cassandra.NewHistoryPersistence. Same for others.